[HW]1072931

liam1909294 opened the pull request:
建議新增結合EX0~EX6所用到技巧的練習題。一次性複習。
Signed-off-by: 1072931 <s1072931@mail.ncyu.edu.tw>

jrjang commented:
Hi @Lan1e ,

1. commit message 不符合規定
  1.1 commit summary 的 [HW] 和 學號 之間要有空白
  1.2 以下描述是寫在 commit message 中
```
建議新增結合EX0~EX6所用到技巧的練習題。一次性複習。
```
2. 麻煩請提交以下描述相關實作
```
建議新增結合EX0~EX6所用到技巧的練習題。一次性複習。
```

Commit:
[HW] 1072931

建議項目:
建議新增結合EX0~EX6所用到技巧的練習題，畢竟中間隔了兩個禮拜，平常也沒在使用，之前的可能也忘得差不多，例如EX2就可結合EX4，讓學生明白版本控制的好處，也可再結合EX5將untracked files清除的作法，把最終決定不上傳的修改取消。類似的練習題。

Signed-off-by: 1072931 <s1072931@mail.ncyu.edu.tw>

liam1909294 commented:
已經修改完成，再請老師過目。

jrjang commented:
Hi @Lan1e ,

作業要求不是給建議, 而是要提交發現的問題與相對應的修改.

例如: 你建議 "建議新增結合EX0~EX6所用到技巧的練習題。一次性複習。"，就要麻煩設計出相對應的練習提並提交
